src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        I am
+        {person.age}
+      </p>
+    )}
+    <p className="Person__partner">
+      {getPartnerName(person)}
+    </p>
+  </section>
+);
+
+function getPartnerName(person) {
+  return (person.partnerName
+    ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+    : 'I am not married'
```

Comment:
[There](https://github.com/mate-academy/react_person/pull/700#discussion_r994939404) means not moving all 2 rows to a separate function.

It should looks like:
```
    <p className="Person__partner">
      ? `${person.partnerName} is my ${getPartner(person)}`
      : 'I am not married'
    </p>
```

Reply:
please, before re-requesting a review, make sure that all corrections from the comments are made